src/Arcus.Observability.Tests.Integration/Serilog/ApplicationInsightsSinkTests.cs

```diff
@@ -197,6 +193,92 @@ public void LogSqlDependency_SinksToApplicationInsights_ResultsInSqlDependencyTe
             }
         }
 
+        [Fact]
+        public void LogEventWithComponentName_SinksToApplicationInsights_ResultsInTelemetryWithComponentName()
+        {
+            // Arrange
+            string componentName = _bogusGenerator.Commerce.ProductName();
+            using (ILoggerFactory loggerFactory = CreateLoggerFactory(config => config.Enrich.WithComponentName(componentName)))
+            {
+                ILogger logger = loggerFactory.CreateLogger<ApplicationInsightsSinkTests>();
+
+                // Act
+                logger.LogInformation("This message will be enriched with a component name");
+
+                // Assert
+                // Hold on till we have agreed on assertion...
+            }
+        }
+
+        [Fact]
+        public void LogEventWithVersion_SinksToApplicationInsights_ResultsInTelemetryWithVersion()
+        {
+            // Arrange
+            using (ILoggerFactory loggerFactory = CreateLoggerFactory(config => config.Enrich.WithVersion()))
+            {
+                ILogger logger = loggerFactory.CreateLogger<ApplicationInsightsSinkTests>();
+
+                // Act
+                logger.LogEvent("Update version");
+
+                // Assert
+                // Hold on till we have agreed on assertion...
+            }
+        }
+
+        [Fact]
+        public void LogEventWithCorrelationInfo_SinksToApplicationInsights_ResultsInTelemetryWithCorrelationInfo()
+        {
+            // Arrange
+            string operationId = $"operation-{Guid.NewGuid()}";
+            string transactionId = $"transaction-{Guid.NewGuid()}";
+            DefaultCorrelationInfoAccessor.Instance.SetCorrelationInfo(new CorrelationInfo(operationId, transactionId));
+            using (ILoggerFactory loggerFactory = CreateLoggerFactory(config => config.Enrich.WithCorrelationInfo()))
+            {
+                ILogger logger = loggerFactory.CreateLogger<ApplicationInsightsSinkTests>();
+
+                // Act
+                logger.LogInformation("This message will be correlated");
```

Comment:
Let's track two messages and ensure that they are linked together.

Reply:
Added...
Is this what you want:
![image](https://user-images.githubusercontent.com/9039753/78526391-2d037a80-77da-11ea-933b-0f1cad277523.png)

Reply:
Yes, thanks!